apps/console/src/features/core/configs/app.ts

```diff
@@ -157,6 +157,7 @@ export class Config {
             gravatarConfig: window["AppUtils"].getConfig().ui.gravatarConfig,
             isClientSecretHashEnabled: window["AppUtils"].getConfig().ui.isClientSecretHashEnabled,
             isGroupAndRoleSeparationEnabled: window["AppUtils"].getConfig().ui.isGroupAndRoleSeparationEnabled,
+            isSignatureValidationCertificateAliasEnabled: window["AppUtils"].getConfig().ui.isSignatureValidationCertificateAliasEnabled,
```

Comment:
Possible character count limit exceeding. Please check and limit line character limit to 120 max.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/1526/commits/69aece1b3e9eb4a72d9cf5496e593dfde17523f9.